### PR TITLE
PR feedback from #21092

### PR DESCRIPTION
**Pull request by esoergel**
Responding to feedback from https://github.com/dimagi/commcare-hq/pull/21092
PRing into that branch
@mkangia

### CODE REVIEW

File: corehq/apps/custom_data_fields/dbaccessors.py [hunk 1]
```diff
@@ -1,7 +1,9 @@
 from __future__ import absolute_import
+from __future__ import unicode_literals
+from corehq.util.quickcache import quickcache
 
 
-from __future__ import unicode_literals
+@quickcache(['domain', 'field_type'], timeout=60)
```
Comment (mkangia) [inline corehq/apps/custom_data_fields/dbaccessors.py]: Whats the argument for caching this? Are custom fields added/updated often enough in a domain to cache this for an hour?

File: corehq/apps/locations/tasks.py [hunk 2]
```diff
@@ -157,9 +157,9 @@ def download_locations_async(domain, download_id, include_consumption, headers_o
 
 @serial_task('{domain}', default_retry_delay=5 * 60, timeout=LOCK_LOCATIONS_TIMEOUT, max_retries=12,
              queue=settings.CELERY_MAIN_QUEUE, ignore_result=False)
-def import_locations_async(domain, file_ref_id, user):
+def import_locations_async(domain, file_ref_id, user_id):
```
Comment (mkangia) [inline corehq/apps/locations/tasks.py]: Good call!

File: corehq/apps/locations/bulk_management.py [hunk 3]
```diff
@@ -52,6 +51,10 @@ def to_boolean(val):
     return False if val == '' else string_to_boolean(val)
 
 
+def strip(val):
+    return val.strip()
+
+
```
Comment (mkangia) [inline corehq/apps/locations/bulk_management.py]: 🥇

**Review by mkangia — APPROVED** [review]

File: corehq/apps/custom_data_fields/dbaccessors.py (hunk 1 above)
Comment (esoergel) [reply corehq/apps/custom_data_fields/dbaccessors.py]: This is a 60 second cache, although now that invalidation is more robust, we could probably lengthen it.  This definition itself is updated very rarely, but it's referenced quite frequently, so caching it should make sense.

Comment (mkangia) [reply corehq/apps/custom_data_fields/dbaccessors.py]: > referenced quite frequently

within the same request? Sorry just trying to understand the intend here. If its cached for 60 seconds only, does not look like its intended for multiple requests.

Comment (esoergel) [reply corehq/apps/custom_data_fields/dbaccessors.py]: I'm not sure whether it's referenced multiple times within the same request, but that's certainly feasible.  It's used in fixture generation, location upload, and editing or adding a location.  60s was just the original cache timeout here: https://github.com/dimagi/commcare-hq/pull/21172/commits/8c2d3c168055b30bc73f8f622bbad2ce519be203#diff-6f967b14b7d77aca82b0459bab5feac8L123

**Pull request merged**